src/Model/Comment.php

```diff
@@ -172,16 +160,9 @@ protected function init() {
 				'type' => function() {
 					return ! empty( $this->comment->comment_type ) ? $this->comment->comment_type : null;
 				},
-				'parent' => function() {
-					$parent = null;
-					if ( ! empty( $this->comment->comment_parent ) ) {
-						$parent_obj = get_comment( $this->comment->comment_parent );
-						if ( is_a( $parent_obj, 'WP_Comment' ) ) {
-							$parent = new Comment( $parent_obj );
-						}
-					}
-					return $parent;
-				},
+				'userId' => function() {
+					return ! empty( $this->comment->user_id ) ? $this->comment->user_id : null;
```

Comment:
Should this default to `0` instead of `null`? We are pretty inconsistent on wether the default is `null`, or the empty value within the scalar type `0`, `''`, etc...

Reply:
well, if the ID is _actually_ 0 we'd want to return 0, but if the ID is 0 or falsey because it's not a user, we should return null, I think...

🤷‍♂️